### PR TITLE
✨ FEAT: LIVE-4457: Notification Center Announcements section in LLM

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### 📝 Description

Notification Center Announcements section in LLM

### ❓ Context

- **Impacted projects**: `live-mobile` `ui:native` 
- **Linked resource(s)**: [LIVE-4457] 

### ✅ Checklist

- [ ] **Test coverage** 
- [X] **Atomic delivery** 
- [X] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-4457]: https://ledgerhq.atlassian.net/browse/LIVE-4457?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/screens/NotificationCenter/helper.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  if (mins < 60) {
+    return [mins, TypeOfTime.minute];
+  }
+
+  if (hours < 24) {
+    return [hours, TypeOfTime.hour];
+  }
+
+  if (days < 24) {
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/screens/NotificationCenter/helper.ts:34]:
```suggestion
  if (days < 7) {
```

File: apps/ledger-live-mobile/src/screens/NotificationCenter/helper.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  const today = new Date().getTime();
+  const fixedTimeStamp = new Date(timestampNew * 1000).getTime();
+  const diff = Math.abs(today - fixedTimeStamp);
+
+  const secs = Math.floor(Math.abs(diff) / 1000);
+  const mins = Math.floor(secs / 60);
+  const hours = Math.floor(mins / 60);
+  const days = Math.floor(hours / 24);
+  const weeks = Math.floor(days / 7);
+  const months = Math.floor(weeks / 12);
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/screens/NotificationCenter/helper.ts:20]:
```suggestion
  const months = Math.floor(weeks / 4);
```

File: apps/ledger-live-mobile/src/screens/NotificationCenter/helper.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  if (weeks < 4) {
+    return [weeks, TypeOfTime.week];
+  }
+
+  if (months < 12) {
+    return [months, TypeOfTime.month];
+  }
+
+  return [
+    Math.abs(new Date(timestampNew).getFullYear() - new Date().getFullYear()),
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/screens/NotificationCenter/helper.ts:47]: Why aren't you using the same formula as above for the other time units ?  Something like `
const years = Math.floor(months / 12);
return [years, TypeOfTime.year]`

File: apps/ledger-live-mobile/src/screens/NotificationCenter/helper.ts (hunk 1 above)
Comment (mcayuelas-ledger) [reply apps/ledger-live-mobile/src/screens/NotificationCenter/helper.ts:34]: Nice catch :)

File: apps/ledger-live-mobile/src/screens/NotificationCenter/helper.ts (hunk 2 above)
Comment (mcayuelas-ledger) [reply apps/ledger-live-mobile/src/screens/NotificationCenter/helper.ts:20]: Nice catch :)

File: apps/ledger-live-mobile/src/components/RootNavigator/NotificationCenterNavigator.tsx [hunk 4]
```diff
@@ -1,103 +1,41 @@
...
 
-import { Flex } from "@ledgerhq/native-ui";
-import styled from "styled-components/native";
-import { TabsContainer } from "@ledgerhq/native-ui/components/Tabs/TemplateTabs";
-import { ChipTab } from "@ledgerhq/native-ui/components/Tabs/Chip";
+import { createStackNavigator } from "@react-navigation/stack";
+import { useTheme } from "styled-components/native";
 import NotificationCenterStatus from "../../screens/NotificationCenter/Status";
-import NotificationCenterNews from "../../screens/NotificationCenter/News";
+import NotificationCenterNews from "../../screens/NotificationCenter/Notifications";
```
[the remark this answers is not in the document]
Comment (mcayuelas-ledger) [reply apps/ledger-live-mobile/src/components/RootNavigator/NotificationCenterNavigator.tsx:7]: Yes we could ! i'll do it

File: apps/ledger-live-mobile/src/screens/WalletCentricAsset/index.tsx [hunk 5]
```diff
@@ -167,6 +168,7 @@ const AssetScreen = ({ route }: NavigationProps) => {
 
   const onPressDismiss = useCallback(() => {
     if (!dynamicContentCard) return;
+    if (!dynamicContentCard.link) return;
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/screens/WalletCentricAsset/index.tsx:171]: If there is no link we can't dismiss a card anymore ?

**Pull request merged**